src/lsptoolshost/commands.ts

```diff
@@ -142,7 +149,29 @@ async function completionComplexEdit(
     }
 
     if (!success) {
-        throw new Error('Failed to make a complex text edit for completion.');
+        const componentName = '[roslyn.client.completionComplexEdit]';
+        const errorMessage = 'Failed to make a complex text edit for completion.';
+        outputChannel.show();
```

Comment:
One thing to consider here - should we put up a toast as well?  We have utils for showing an error message.  Should only do that though if we want to bring attention to it and don't think it will be popping up all the time.

Reply:
good point, we probably can hook up the toast, given this is a failure from user explicitly invoked action. Will add in a follow up PR